nilearn/_utils/helpers.py

```diff
@@ -126,19 +130,24 @@ def remove_parameters(removed_params,
         For informational purpose in the warning text.
 
     """
+
     def _remove_params(func):
         @functools.wraps(func)
         def wrapper(*args, **kwargs):
             found = set(removed_params).intersection(kwargs)
             if found:
-                message = (f'Parameter(s) {", ".join(found)} '
-                           f'will be removed in version {end_version}; '
-                           f'{reason}')
-                warnings.warn(category=DeprecationWarning,
-                              message=message,
-                              stacklevel=3)
+                message = (
+                    f'Parameter(s) {", ".join(found)} '
+                    f"will be removed in version {end_version}; "
+                    f"{reason}"
+                )
+                warnings.warn(
+                    category=DeprecationWarning, message=message, stacklevel=3
+                )
             return func(*args, **kwargs)
+
         return wrapper
+
```

Comment:
Sorry for the messy diff. Looks like this module wasn't yet blackified.
All changes above this are just blackification.

Reply:
that's fair: I actually closed the very old PR where this file was also involved and why we were not letting black work on it

can I let you update the black and pre-commit config to let black validate this file in CI:

just remove it from the exclude 
- here: https://github.com/nilearn/nilearn/blob/e384ebc7c5a5a080b476efdb4a9aa8e9af57eff0/.pre-commit-config.yaml#L75
- and here: https://github.com/nilearn/nilearn/blob/e384ebc7c5a5a080b476efdb4a9aa8e9af57eff0/pyproject.toml#L108